.github/workflows/commitlint.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - uses: wagoid/commitlint-github-action@v1.6.0
+      - uses: wagoid/commitlint-github-action@master
```

Comment:
Is there a release we could use? Master typically is just less stable.

Reply:
yes, we have